source/application/controllers/admin/voucherserie_export.php

```diff
@@ -113,7 +113,12 @@ protected function _getExportFileName()
      */
     protected function _getExportFilePath()
     {
-        return $this->getConfig()->getConfigParam('sShopDir') . "/export/" . $this->_getExportFileName();
+        $sExportDir = $this->getConfig()->getConfigParam( 'sShopDir' ) . "/export/";
+        if (!file_exists($sExportDir) || !is_dir($sExportDir)) {
```

Comment:
What is the reason to use ``!file_exists($sExportDir)`` here. Maybe it would be enough ``!is_dir($sExportDir)``